packages/extension-ui/src/Popup/Accounts/NewAccount.tsx

```diff
@@ -29,9 +37,12 @@ const ButtonsGroup = styled.div`
   gap: 8px;
   padding-bottom: 0px;
   position: absolute;
-  bottom: 16px;
+  bottom: 7px;
   left: 0px;
   right: 0px;
+  height: 56px;
+  backdrop-filter: blur(10px);
```

Comment:
Is this crucial? Backdrop-filer usually impacts performance and I would prefer someone tested performance after this change...